apps/condo/domains/common/utils/pdf.ts

```diff
@@ -81,3 +83,28 @@ export const createPdf: ICreatePdf = (options) => {
         return doc.save(fileName, { returnPromise: true })
     })
 }
+
+export const createPdfWithPageBreaks: ICreatePdfWithPageBreaks = (options) => {
+    const { element, fileName } = options
+    const elementWidth = element.clientWidth
+    const elementHeight = element.clientHeight
+    const topLeftMargin = 15
```

Comment:
get from options

Reply:
Did you mean to get element properties from options?